### PR TITLE
Project media query

### PR DESCRIPTION
**Pull request by jedwardmook**
Trying out the new commit system for the hero project >= 900px.

### CODE REVIEW

File: src/components/ProjectImagesViewer.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ function ProjectImagesViewer({ styles, images, isViewerOpen, setIsViewerOpen })
         </div>
         <div className={styles['image-viewer-hero-container']}>
           <img className={styles['image-viewer-hero-image']} src={heroImage.link} alt={heroImage.description}/>
-          <p>{heroImage.description}</p>
+          <p className={styles['image-viewer-hero-discription']}>{heroImage.description}</p>
```
Comment (inkbeard) [inline src/components/ProjectImagesViewer.js]: nit: spelling:

```suggestion
          <p className={styles['image-viewer-hero-description']}>{heroImage.description}</p>
```

File: src/components/projects.module.css [hunk 2]
```diff
@@ -179,12 +180,55 @@
   width: 60vw;
 }
 
+.image-viewer-hero-discription {
```
Comment (inkbeard) [inline src/components/projects.module.css]: nit: spelling:

```suggestion
.image-viewer-hero-description {
```

**Review by inkbeard — COMMENTED** [review]
noticed a spelling issue

File: src/components/ProjectImagesViewer.js (hunk 1 above)
Comment (jedwardmook) [reply src/components/ProjectImagesViewer.js]: Lordy, I looked at that and thought that doesn't look right. Have the correct one right next to it. Sheesh

File: src/components/projects.module.css [hunk 3]
```diff
@@ -122,6 +122,7 @@
   margin-left: .5vw;
   margin-right: 2vw;
   font-size: 2vw;
+  margin-bottom: -.75vw;
```
Comment (inkbeard) [inline src/components/projects.module.css:125]: Why is vw (vertical width) being used for a margin bottom versus vh (vertical height)?

**Review by inkbeard — COMMENTED** [review]
I don't think it's hurting anything per se, but I would look into rem/em/vw/vh usage. To be honest, I've rarely used `vh` outside of making something 100vh, but doing some research, I don't see `vw`/`vh` really being used for font-sizes like you have here. For font-sizing, it's suggested to use `em` and `rem`.

File: src/components/projects.module.css (hunk 3 above)
Comment (jedwardmook) [reply src/components/projects.module.css:125]: Yeah, I probably went a little nuts with the vw. I am just trying to figure out the responsive design structure. I think you're totally right about rem and em as the designer of the site I am doing freelance said he never heard of a vw. As for the using vw or vh, I was just being consistent and thinking more of it as a measurement over anything else

**Review by inkbeard — APPROVED** [review]
for future PR:
- switch to REMs for font-sizes
- scroll to hero on thumbnail click

**Pull request merged**